benchmarks/src/main.rs

```diff
@@ -141,304 +193,334 @@ const WRITE_BENCHES: &[&'static str] = &[
     GRIDFS_MULTI_UPLOAD_BENCH,
 ];
 
-async fn run_benchmarks(uri: &str, more_info: bool, ids: &[bool]) -> Result<CompositeScore> {
+const MAX_ID: usize = 17;
```

Comment:
Rather than a hand-maintained constant, this could be a terminal arm of the `BenchmarkId` enum.

Reply:
I updated this to be defined in terms of the enum, but I stopped short of adding a new, terminal variant (if that's what you were suggesting) so that we wouldn't have to handle it everywhere we match on `BenchmarkId`. Instead, I added an assertion that `MAX_ID` is in fact the max element in the enum, which should ensure this doesn't accidentally become out of sync.

Reply:
Yup, a terminal variant is what I was thinking of, but your way covers it nicely as well.